omnibus-arbitrum-mainnet.toml

```diff
@@ -369,7 +369,8 @@ pyth_feed_id_blur = "0x856aac602516addee497edf6f50d39e8c95ae5fb0da1ed434a8c2ab9c
 pyth_feed_id_bnb = "0x2f95862b045670cd22bee3114c39763a4a08beeb663b145d283c31d7d1101c4f"
 pyth_feed_id_bome = "0x30e4780570973e438fdb3f1b7ad22618b2fc7333b65c7853a7ca144c39052f7a"
 pyth_feed_id_bonk = "0x72b021217ca3fe68922a19aaf990109cb9d84e9ad004b4d2025ad6f529314419"
-pyth_feed_id_btc = "0x56a3121958b01f99fdc4e1fd01e81050602c7ace3a571918bb55c6a96657cca9"
+pyth_feed_id_btc = "0xc9d8b075a5c69303365ae23633d4e085199bf5c520a3b90fed1322a0342ffc33"    # Note, this is the wBTC feed id not the pure BTC feed Id
```

Comment:
do we need this?

Reply:
yeah ill remove this change, must have slipped through,  from the original branch.... this needs to be done, but manually so that we dont have 2 settle strategies, thanks for catching